golang/internal/helper/docker/container_test.go

```diff
@@ -0,0 +1,185 @@
+//go:build unit
+// +build unit
+
+package docker_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+	"github.com/rs/zerolog/log"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"github.com/thanhpk/randstr"
+
+	dockerHelper "github.com/dyrector-io/dyrectorio/golang/internal/helper/docker"
+	containerbuilder "github.com/dyrector-io/dyrectorio/golang/pkg/builder/container"
+)
+
+type DockerContainerHelperTestSuite struct {
+	suite.Suite
+	testContainers []*containerbuilder.DockerContainerBuilder
+	dockerClient   client.Client
+	ctx            context.Context
+	prefix         string
+	containerNames []string
+}
+
+const (
+	nginxImage   = "docker.io/library/nginx:latest"
+	prefixLength = 8
+)
+
+// Runs before everything
+func (testsuite *DockerContainerHelperTestSuite) SetupSuite() {
```

Comment:
Naming these BeforeAll or something like that, but @szszigeti can help with the naming convention, IMHO.

Reply:
AFAIK, these function names have to be named like that. This is how this testing suite works. :shrug:

Reply:
Hmm, okey than sorry.